js-pkg/driftdb/src/index.ts

```diff
@@ -115,18 +122,25 @@ export class DbConnection {
             console.log("Connection closed, attempting to reconnect...")
 
             this.reconnectLoopHandle = setTimeout(() => {
-                this.connect(dbUrl)
+                this.connect(dbUrl, binary)
             }, 1000)
         }
 
         this.connection.onmessage = (event) => {
-            const message: MessageFromDb = JSON.parse(event.data as string)
+            let message: MessageFromDb
+            if (event.data instanceof ArrayBuffer) {
+                message = CBOR.decode(event.data as any)
+            } else {
+                message = JSON.parse(event.data as string)
```

Comment:
Do you need the `as any` and `as string` here?

Reply:
done